channels/sa.m3u

```diff
@@ -17,6 +15,8 @@ http://m.live.net.sa:1935/live/quran/playlist.m3u8
 https://cdnamd-hls-globecast.akamaized.net/live/ramdisk/saudi_quran/hls1/saudi_quran.m3u8
 #EXTINF:-1 tvg-id="AlQuranAlKareemTV.sa" tvg-name="Al Quran Al Kareem TV" tvg-country="SA" tvg-language="Arabic" tvg-logo="https://i.imgur.com/0aMNwQa.png" group-title="Religious",Al Quran Al Kareem TV (Mecca) (1080p)
 https://iptv--iptv.repl.co/Arabic/Quran_TV/
+#EXTINF:-1 tvg-id="AlQuranAlKareemTV.sa" tvg-name="Al Quran Al Kareem TV" tvg-country="SA" tvg-language="Arabic" tvg-logo="https://i.imgur.com/0aMNwQa.png" group-title="Religious",Al Quran Al Kareem TV (Mecca)
+https://5b18be6964c2f.streamlock.net:1935/live/_definst_/quran/playlist.m3u8
```

Comment:
Returns playlist but can't confirm it working

Reply:
Working https://streamtest.in/logs/https-5ab29cc78f681-streamlock-net-makkahtv-livestream-playlist-m3u8